docs/verifiable-document/document-template.md

```diff
@@ -4,8 +4,228 @@ title: Deploying Document Renderer
+```
+
+*Note: The `default` template is a special template that will be used as the fallback template when a template of the correct name cannot be found.*
+
+### Multiple views for a Template
+
+An OA document may have multiple views, each of them rendered in a separate tabs. The views are defined in the `templates` array in `index.tsx` of the template folder. Each of these views can have its own id, label and template code.
+
+For our Certificate of Completion, we will only use a single view. So we will remove any additional views and rename the first view's label as "Certificate".
+
+Replace the code in `src/templates/customTemplate/index.tsx` with the following:
+
+```js
+import { CustomTemplate } from "./customTemplate";
+
+export const templates = [
+  {
+    id: "certificate",
+    label: "Certificate",
+    template: CustomTemplate
+  }
+];
+```
+
+### Developing the COC Template View
+
+Finally, once all the components have been wired up, we may proceed to style our Certificate of Completion.
+
+To change how the data is being renderered, we simply create a React component that takes in the unwrapped document in the `document` props and render the corresponding HTML code.
+
+For our COC, we will simply display the following text:
+
+```text
+OpenAttestation Tutorial Certificate of Completion
+awarded to
+John Doe
+```
+
+To do so, you may update the template code in `src/templates/customTemplate/customTemplate.tsx` to the following:
+
+```js
+import React, { FunctionComponent } from "react";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import { css } from "@emotion/core";
+import { CustomTemplateCertificate } from "../sample";
+import { PrintWatermark } from "../../core/PrintWatermark";
+
+const containerStyle = css`
+  background-color: #324353;
+  color: #ffffff;
+  padding: 15px;
+  margin: auto;
+  width: 80%;
+  text-align: center;
+`;
+
+export const CustomTemplate: FunctionComponent<
+  TemplateProps<CustomTemplateCertificate> & { className?: string }
+> = ({ document, className = "" }) => {
+  return (
+    <div css={containerStyle} className={className} id="custom-template">
+      <PrintWatermark />
+      <h1>{document.name}</h1>
+      <div>awarded to</div>
+      <h2>{document.recipient.name}</h2>
+    </div>
+  );
+};
+```
+
+### Completion
+
+![Completed Story Book View](/docs/verifiable-document/document-template/completed-storybook.png)
+
+Once that template code has been updated, you will see that the storybook component has been updated to reflect the change.
+
+Now, your document renderer is ready to be built and deployed online.
+
+## Deploying Document Renderer
+
+### Push code to github.com
+
+Create a new repository in github and the code to the new repository. For a step-by-step guide to import source code to github, you may read [this guide](https://help.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line).
```

Comment:
and `push` the code

Reply:
Where do you want this? I imagine people knowing that, and if they dont the guide will teach them how.

Reply:
fixed